### PR TITLE
Add docs for the SolarAbund class

### PR DESCRIPTION
**Pull request by nhmc**
Also some other minor changes.

### CODE REVIEW

File: linetools/abund/elements.py [hunk 1]
```diff
@@ -290,7 +291,7 @@ def __str__(self):
                                      self.abundance * 100)
 
     def __repr__(self):
-        return "Isotope(%s, %s, %s)" % (
+        return "<Isotope(%s, %s, %s)>" % (
```
Comment (profxj) [inline linetools/abund/elements.py]: maybe change to {:s} format, if you can

**Comment by profxj** [comment]
Looks good.  Pass the test and merge.

File: linetools/abund/elements.py (hunk 1 above)
Comment (nhmc) [inline linetools/abund/elements.py]: There are 30+ strings to do in that module, so I might skip this for now.

**Comment by nhmc** [comment]
Merging.

**Pull request merged**